### ISSUE TITLE
Submission status is "error"... 

### ISSUE THREAD

**Issue by dnil** [Clinical-Genomics/scout#3790]

**Comment by dnil** [comment]
No, but you shouldn't need to - the transcript name is an integral part of the HGVS descriptor!

**Comment by northwestwitch** [comment]
> part of the HGVS descriptor

Right!

**Comment by northwestwitch** [comment]
The thing is here then:  [   clinvar_var["ref_seq"] = tx_hgvs.split(":")[0]](https://github.com/Clinical-Genomics/scout/blob/b0cfc8795392ed7e1b223eeaa5ad5590fc6e8892/scout/server/blueprints/clinvar/controllers.py#L214)

It's splitting hgvs into 2 things to pass them separately to preclinvar

**Comment by northwestwitch** [comment]
I'll write a fix!

**Comment by northwestwitch** [comment]
And I'll move this issue back to preClinVar
